lab_1_classify_by_unigrams/start.py

```diff
@@ -1,19 +1,45 @@
 """
 Language detection starter
 """
+from lab_1_classify_by_unigrams.main import (collect_profiles,
+                                             create_language_profile,
+                                            detect_language_advanced,
+                                            print_report)
 
 
 def main() -> None:
     """
     Launches an implementation
     """
-    with open("assets/texts/en.txt", "r", encoding="utf-8") as file_to_read_en:
-        en_text = file_to_read_en.read()
-    with open("assets/texts/de.txt", "r", encoding="utf-8") as file_to_read_de:
-        de_text = file_to_read_de.read()
+    paths_to_profiles = ['assets/profiles/en.json', 'assets/profiles/es.json',
+                         'assets/profiles/fr.json','assets/profiles/it.json',
+                         'assets/profiles/ru.json','assets/profiles/tr.json']
+
+    # with open("assets/texts/en.txt", "r", encoding="utf-8") as file_to_read_en:
```

Comment:
what is it? revert

Reply:
What exactly do you mean? Should I delete code with # and remain only code for 10 score?